src/renderer/components/Keybindings/index.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useContext } from 'react';
+import Styles from './styles.module.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+import { Action, useKeybindings } from 'renderer/hooks/useKeybindings';
+import { useHotkeyRecorder } from 'renderer/hooks/useHotkeysRecorder';
+import Modal from '../Modal';
+import { ChildWindowContext } from '../ChildWindow';
+
+export default function Keybindings() {
+  const { keybindings, setKeybinding } = useKeybindings();
+
+  const [recordingAction, setRecordingAction] = React.useState<Action | null>();
+
+  const [editModalOpen, setEditModalOpen] = React.useState(false);
+
+  const onSave = (keybinding: string) => {
+    if (!recordingAction) return;
+    setKeybinding(recordingAction, keybinding);
+    setEditModalOpen(false);
+  };
+
+  if (!keybindings) return null;
+
+  return (
+    <React.Fragment>
+      <Modal
```

Comment:
What's the point of putting it in the modal when it is primary used ChildWindow. The purpose of ChildWindow is to mimic the modal already.

Reply:
Okay. I understand why. Let me check further. Still have problem running your code lol

Reply:
Dont forget to reset the indexedDB otherwise it wont work, also run npm install too